### ISSUE TITLE
About accuracy

### ISSUE THREAD

**Issue by Hrqingqing** [akamaster/pytorch_resnet_cifar10#28]
Hello, thanks for your contribution. I run your code, but I can't obtain the result that you reported. Such as resnet20 in Cifar10, the accuracy is only 88.73. I hope you can give me some help.

**Comment by akamaster** [comment]
Yeah, recently I have had these comments. For example see #18, [and specifically my comment within](https://github.com/akamaster/pytorch_resnet_cifar10/issues/18#issuecomment-605314203) Essentially, the issue is in PyTorch version, starting from some (1.3?) version something has changed in optimizers. Please run with PyTorch 1.2, it should give your the necessary result.

**Comment by Hrqingqing** [comment]
> Yeah, recently I have had these comments. For example see #18, [and specifically my comment within](https://github.com/akamaster/pytorch_resnet_cifar10/issues/18#issuecomment-605314203) Essentially, the issue is in PyTorch version, starting from some (1.3?) version something has changed in optimizers. Please run with PyTorch 1.2, it should give your the necessary result.
OK, thanks for your reply! And I would like to ask if you have the classification results on CIFAR100?
